spec/menu_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'menu'
+
+describe Menu do
+
+    menu = Menu.new
+    item1 = Item.new(1, "Eggs and Turnips", 10)
```

Comment:
When testing a class, you should try to avoid relying on another class. Doubles would work here but it looks like you'd have to make a few..., might be a better way?